lib/Distribution/Opensuse/Tumbleweed.pm

```diff
@@ -10,22 +10,22 @@
 # Summary: The class represents Tumbleweed distribution and provides access to
 # its features.
 
-# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
 
 package Distribution::Opensuse::Tumbleweed;
 use strict;
 use warnings FATAL => 'all';
 use parent 'susedistribution';
 use Installation::Partitioner::LibstorageNG::GuidedSetupController;
-use Installation::Partitioner::LibstorageNG::v4::ExpertPartitionerController;
+use Installation::Partitioner::LibstorageNG::v4::_3::ExpertPartitionerController;
```

Comment:
I'm not sure about the underscore, specially for future to be valid in CPAN, but it is not a big problem now. I see that there they are using something like: `Net::Twitter::Lite::API::V1_1` so the name convention is to have always capital letter first. You can decide if makes sense at this point.

Reply:
_3 is a valid variable name. I decided to have it as a sub-directory, we can move it to `V4_3` I guess, but then we need to rename all other modules. Also Alex named packages like `Distribution::Sle::15sp0`, which is basically against standards as module names follow same restrictions as variable names and those should start with the letter.
But let me at least move it to `v4_3` directory, and somewhere later we can capitalize it.